omod/pom.xml

```diff
@@ -15,7 +15,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>legacyui</artifactId>
-		<version>1.3.3</version>
+		<version>1.4.0-SNAPSHOT</version>
```

Comment:
Since when did this change from https://github.com/openmrs/openmrs-module-legacyui/blob/master/omod/pom.xml#L18